src/main/kotlin/io/snyk/plugin/ui/SnykSettingsDialog.kt

```diff
@@ -463,6 +465,15 @@ class SnykSettingsDialog(
             )
         )
 
+        cliBaseDownloadUrlTextField.toolTipText = "The default URL is https://static.snyk.io. " +
```

Comment:
question: do we really want to mention this here? From a maintenance perspective, I think it would be better to leave this to the user-docs.